new-frontiers-14/frontier-station-14
# Mail Tracking (DeltaV)

## About the PR
Port https://github.com/DeltaV-Station/Delta-v/pull/1472 from DeltaV

## Why / Balance
Allow SR and mail to track mail profits

## How to test
Spawn as mail, make sure mail box is on, run ``mailnow`` and open mail, look in mail app in PDA.

## Media

## Breaking changes
N/A

**Changelog**
:cl:
- add: The mail courier and SR now have a new program in their PDA for tracking mail delivery performance, including earnings and percent of packages opened, damaged, or expired.
